packages/ketcher-macromolecules/src/hooks/useIDTAliasesTextForPreset.ts

```diff
@@ -0,0 +1,44 @@
+import { IKetIdtAliases } from 'ketcher-core';
+import { useMemo } from 'react';
+import { PresetPosition } from 'state/common';
+
+type Props = {
+  presetName: string | undefined;
+  position: PresetPosition;
+  idtAliases: IKetIdtAliases | undefined;
+};
+
+const useIDTAliasesTextForPreset = ({
+  presetName,
+  position,
+  idtAliases,
+}: Props) => {
+  return useMemo(() => {
+    if (!presetName || !idtAliases) {
+      return null;
+    }
+
+    if (presetName.includes('MOE')) {
```

Comment:
Why rules applied only for MOE? I believe it should be for all presets

Reply:
There is the following requirement for these specific presets:
<img width="572" alt="image" src="https://github.com/user-attachments/assets/1c2fbc6f-a782-43cd-9eb6-3a69e9214056">
All the other presets should just have their base both on canvas and in the library. I don't like handling it by name though, maybe we could handle it in a better manner later